web/crux-ui/src/server/registry-api/unchecked-api-client.ts

```diff
@@ -0,0 +1,19 @@
+import { RegistryImageTags } from '@app/models'
+import { RegistryApiClient } from './registry-api-client'
+
+class UncheckedApiClient implements RegistryApiClient {
```

Comment:
why is this exists? We're never going to receive a search ws message to an unchecked registry, right?

Reply:
True, I handle it as an error instead since it shouldn't have an API.